Resources/Prototypes/Recipes/Lathes/medical.yml

```
@@ -188,7 +188,7 @@
   result: Jug
   completetime: 4
   materials:
-    Plastic: 400
+    Plastic: 100 # Frontier
```

```suggestion
    Plastic: 100 # Frontier: 400<100
```

Toss in applyMaterialDiscount to false or set the cost to 200 and this seems fine - think it recycles to 50 plastic anyways.

Five additional chem dispenser slots for jugs and five or ten more empty $100 jugs in the ChemVend also seems reasonable to me.

For what it's worth, plastic should be more abundant than ever on wrecks.